src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
+
+      <div class="calendar-day"></div>
```

Comment:
It seems like you are trying to create a calendar with 30 days. Instead of creating each day manually, you can use JavaScript to generate the days dynamically. This will make your code more maintainable and flexible. If you need to change the number of days, you can easily do it in the JavaScript code.